src/portfolio/Risk.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import CheckIcon from '@material-ui/icons/Check';
+import WarningIcon from '@material-ui/icons/Warning';
+import PriorityHighIcon from '@material-ui/icons/PriorityHigh';
+
+/** Icons to be used in the risk section */
+const exclamationIcon = (color: string) => (
+  <PriorityHighIcon
+    style={{ color, width: '100%', height: '100%' }}
+    aria-label="exclamationIcon"
+  />
+);
+
+const warningIcon = (color: string) => (
+  <WarningIcon
+    style={{ color, width: '100%', height: '100%' }}
+    aria-label="warningIcon"
+  />
+);
+
+const checkIcon = (color: string) => (
+  <CheckIcon
+    style={{ color, width: '100%', height: '100%' }}
+    aria-label="checkIcon"
+  />
+);
+
+export type RiskBundle = {
+  count: number;
+  riskColor: string;
+  riskIcon: JSX.Element;
+  warnings: string[];
+};
+
+/**
+ * @param type - The type of risk to be evaluated
+ * @param count - The number of of countries, segments or currencies
+ * @param threshold1 - The first threshold: Up to this number (exclusive)
+ * the text color will be red and the icon will be an exlamation mark
+ * @param threshold2 - The second threshold: Up to this number (exclusive)
+ * the text color will be yellow and the icon will be an warning sign.
+ * Any thing past threshold2 will be green
+ * @param red - The color for red
```

Comment:
I don't know if it makes sense to forward these color strings as parameters instead of putting them inside of here. They will always be the same. I get that the way it is done know you just have to change the colors in DetailsMainRisk but I still think it's a bit weird like this.

Reply:
The problem was I couldn't use the theme because this is not a react component. I'd have to import the theme directly from App.tsx but that leads to a cyclical dependency. Is there a way of accessing the theme outside of react components?